Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Models/GeneratedMethodModelBuilder.cs

```diff
@@ -215,6 +215,11 @@ private static IList<ParameterModel> BuildParameters(IMethodSymbol lambdaMethodS
                     }
                     else
                     {
+                        // Somehow, Roslyn doesn't include @ character when using with reserved keyword. Add a check just in case Roslyn fix is implemented in later versions.
```

Comment:
i didnt get what this comment means. why do we have to have this if statement?

Reply:
For this customer's scenario, they use `@event` as parameter name. When code reaches here, Roslyn doesn't return its name as `@event`, it returns `event`. This causes issue as mentioned by customer. This condition will handle scenario if Microsoft decides to fix it and return parameter name as set by user. `@` character can only be present at the beginning of the identifier.

Reply:
`@ character can only be present at the beginning of the identifier.` - ah okay. this is what i didnt understand at first. Makes sense then